src/node/createSetupServer.ts

```diff
@@ -111,6 +115,8 @@ export function createSetupServer(...interceptors: Interceptor[]) {
             mockedRequest.cookies = cookieUtils.parse(requestCookieString)
           }
 
+          emitter.emit('request:start', mockedRequest)
```

Comment:
Moving the emit of the `request:start` even so that its listener would accept an already transformed request (all cookies attached, any other coercions and transformations finished).